packages/node_modules/@cerebral/http/src/HttpProviderError.js

```diff
@@ -1,11 +1,11 @@
 export default class HttpProviderError extends Error {
-  constructor(status, headers, body, message = null, isAborted = false) {
+  constructor(status, headers, result, message = null, isAborted = false) {
     super()
     this.name = 'HttpProviderError'
     this.message = message
     this.status = status
     this.headers = headers
-    this.body = body
+    this.result = result
```

Comment:
I think "body" would be better. I was scratching my head as to what a "result" could be when obviously it failed (since there is an error)... Or call it "response"...